modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase

```diff
@@ -26,6 +26,37 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test DownloadSampleFileOfTheCustomObject {
+		task ("Given a custom object is published") {
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "student",
+				en_US_plural_label = "students",
+				name = "Student",
+				requiredStringFieldName = "name");
+		}
+
+		task ("And Given object entry is created") {
+			ObjectDefinitionAPI.createObjectEntryWithName(
+				en_US_plural_label = "students",
+				name = "Jessica");
+		}
+
+		task ("And Given I create an import file task with custom object as the entity type") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoImport();
+		}
+
+		task ("When I download a sample file of this entity") {
+			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
```

Comment:
I think this should be in the upper step "And Given I create an import file task with custom object as the entity type"

Reply:
Thanks, i will update

Reply:
And just a suggestion, please make sure don't modify the test's step if no need

Reply:
Done, thanks guys